packages/tests/package.json

```
@@ -45,6 +47,18 @@
       "dependencies": [
         "build"
       ]
```

```suggestion
      "dependencies": [
        "test:dev",
        "test:prod"
      ]
```

(And remove "command", I couldn't put that in the suggestion)

---

(this is optional, just a neater way to write the same thing)

---

will do!

---

done!

---

Make sure to also remove the command, or else it will run everything twice.

---

I still see the command